.github/workflows/build_and_test.yml

```diff
@@ -70,6 +86,12 @@ jobs:
         aws-region: ${{ secrets.AWS_REGION }}
     - uses: ros-tooling/action-cloudwatch-metrics@0.0.4
       with:
+        metric-dimensions: >-
+          [
+            { "Name": "github.event_name", "Value": "${{ github.event_name }}" },
+            { "Name": "github.ref", "Value": "${{ matrix.build_type }}" },
```

Comment:
Will this work? Isnt the matrix isolated to "build_and_test"?

Reply:
Nope